README.md

```diff
@@ -63,36 +24,29 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [vet-clinic-database] <a name="vet-clinic"></a>
 
 > Describe your project in 1 or 2 sentences.
```

Comment:
- [optional] Please kindly look through your readme file patiently and remove all the placeholders. Placeholders are meant to tell us what to do with each header. ℹ️ The same is applicable to the template text in the license section of your readme file. 👍 

<img width="665" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/228204052-0b03f1ac-a89d-4bd9-891b-d473e3bbc8aa.png">